src/features/references/editor/__tests__/ReferencesTableView.test.tsx

```diff
@@ -117,24 +118,21 @@ describe('ReferencesTableView component', () => {
   it(`should not emit ${'refstudio://references/remove' as RefStudioEventName} with NO selections`, async () => {
     const { user } = setupWithJotaiProvider(<ReferencesTableView />, store);
 
-    const actionsMenu = screen.getByTestId('actions-menu');
-    await user.hover(actionsMenu);
-    const removeBtn = within(actionsMenu).getByText('Remove');
+    const removeBtn = screen.getByText('Remove');
     await user.click(removeBtn);
 
+    expect(removeBtn).toBeDisabled();
     expect(removeBtn).toHaveAttribute('aria-disabled', 'true');
     expect(vi.mocked(emitEvent)).not.toHaveBeenCalled();
   });
 
-  // Note: This test can't run because there is a weird issue when clicking remove ("TypeError: Cannot read properties of undefined (reading 'contains')")
-  // https://github.com/ag-grid/ag-grid/issues/6179
-  it.skip(`should emit ${'refstudio://references/remove' as RefStudioEventName} with SOME selections`, async () => {
```

Comment:
I don't know what was the issue and what fixed it, but I tried to unskip the test and it passed! 🎉